code/modules/mob/living/simple_animal/abnormality/aleph/silent_orchestra.dm

```diff
@@ -114,7 +114,7 @@
 						playsound(get_turf(H), 'sound/abnormalities/silentorchestra/headbomb.ogg', 50, 1)
 				SLEEP_CHECK_DEATH(4 SECONDS)
 				animate(src, alpha = 0, time = 2 SECONDS)
-				QDEL_IN(src, 2 SECONDS)
+				addtimer(CALLBACK(src, death()), 2 SECONDS) // Qdel breaks anything that needs him to DIE
```

Comment:
It does not break anything at all..?

Reply:
You might be referring to signals that only listen to death, in which case, just add a signal listening to Destroy() proc.